src/main/java/com/mixpanel/android/mpmetrics/MPConfig.java

```diff
@@ -208,7 +212,16 @@ public int getBulkUploadLimit() {
 
     // Target max milliseconds between flushes. This is advisory.
     public int getFlushInterval() {
-        return mFlushInterval;
+        return getFlushInterval(null);
```

Comment:
How often do we call this function? It's running on the mobile device. So, if it's quite often, I'd prefer to return mFlushInterval. Otherwise, I'm fine with it.